llvm/tools/llvm-profgen/PerfReader.cpp

```diff
@@ -1152,15 +1226,35 @@ void PerfScriptReader::parseAndAggregateTrace() {
 // A LBR sample is like:
 // 40062f 0x5c6313f/0x5c63170/P/-/-/0  0x5c630e7/0x5c63130/P/-/-/0 ...
 // A heuristic for fast detection by checking whether a
-// leading "  0x" and the '/' exist.
+// leading "0x" and the '/' exist.
 bool PerfScriptReader::isLBRSample(StringRef Line) {
-  // Skip the leading instruction pointer
+  // Skip the leading instruction pointer, and maybe PID
+
   SmallVector<StringRef, 32> Records;
-  Line.trim().split(Records, " ", 2, false);
-  if (Records.size() < 2)
-    return false;
-  if (Records[1].starts_with("0x") && Records[1].contains('/'))
+
+  // Trim off the PID (and maybe part/all of the IP)
```

Comment:
Could you show an example of the LBR line with PID? And maybe update the comment with it as well?